index.html

```diff
@@ -1,26 +1,27 @@
 <!doctype html>
 <html>
   <head>
-    <link rel='stylesheet' href='styles.css'>
+    <title>John's page</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
   </head>
   <body>
     <header>
       <h1>The Header</h1>
     </header>
-    <aside>
+    <aside class="aside1">
```

Comment:
A semantic tag would be something like `<main>`. For this question, you need to use a new HTML tag, not a new class.